api/platform/platform_controller.go

```diff
@@ -136,6 +143,7 @@ func (c *Controller) patchPlatform(request *web.Request) (*web.Response, error)
 		return nil, errDecode
 	}
 	newPlatform.ID = platformID
+	newPlatform.CreatedAt = time.Time{}
 	newPlatform.UpdatedAt = time.Now().UTC()
 	platformStorage := c.PlatformStorage
```

Comment:
Why extract the storage to a variable? I'd rather use ```c.PlatformStorage``` directly on the next line for the sake of consistency throughout the file.

Reply:
It is used in line 157 as well